docs/getting_started/tutorial-hyperbolic-discounting.ipynb

```diff
@@ -0,0 +1,1402 @@
+{
```

Comment:
<p>Some notes to align the style with rest of the documentation:</p><ul><li><code>n_points</code></li><li> <code>grid_start = 0</code> . No need to make it a float. Python is not typed.</li><li><code>list()</code>  should be <code>[]</code> . Same goes for dictionaries. It is faster (although negligible), but it is more pythonic.</li><li>There is no need to save characters in <code>rslts</code>  and <code>rslt</code> . Call it <code>mean_max_exp_fishing</code>  or something else more descriptive.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/347/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>